streaming/streaming.go

```diff
@@ -445,16 +445,22 @@ func LaunchDataflowJob(ctx context.Context, targetProfile profiles.TargetProfile
 	} else {
 		dataflowHostProjectId = dataflowCfg.HostProjectId
 	}
+
 	dataflowSubnetwork := ""
+
+	// if custom network is not selected, use public IP. Typical internal testing flow
```

Comment:
nit: golang comment style should be full sentences aka start with capital letter and end with full stop.